contracts/v2/PolygonZkEVMBridgeV2.sol

```diff
@@ -726,6 +734,33 @@ contract PolygonZkEVMBridgeV2 is
             ];
     }
 
+    /**
+     * @notice Set the address of a wrapper using the token information if already exist
+     * @dev This function is used to allow any existing token to be mapped with
+     *      origin token. Wrapper contract should handle mint/burn of the existing token.
+     * @param originNetwork Origin network
+     * @param originTokenAddress Origin token address, 0 address is reserved for ether
+     * @param wrappedTokenAddress Arbitary contract that implements TokenWrapped interface
+     */
+    function setTokenWrappedAddress(
+        uint32 originNetwork,
+        address originTokenAddress,
+        address wrappedTokenAddress,
```

Comment:
i think this is unnecessary - we just want to set existingTokenAddress
(what would even be here?)